packages/cells/src/widget.ts

```diff
@@ -957,6 +957,7 @@ export class CodeCell extends Cell<ICodeCellModel> {
     const model = this.model;
     this.maxNumberOutputs = options.maxNumberOutputs;
 
+    this.node.setAttribute('role', 'textbox');
```

Comment:
1. I am not convinced whether this should be added on `jp-Cell` node which includes the execution counter and outputs:
    ![Screenshot from 2023-01-02 14-18-48](https://user-images.githubusercontent.com/5832902/210243629-de19fc9c-06ac-4af3-be2d-1297acb889ce.png)
    setting it further down on `jp-InputArea-editor`/`jp-Editor` seems more appropriate to me but I may be wrong here:
     ![Screenshot from 2023-01-02 14-19-25](https://user-images.githubusercontent.com/5832902/210243622-cec88e4e-49cd-4cab-8143-9c404d503b9d.png)
2. The issue says *Codemirror: missing aria-label to div with role* but I do see aria-role on CodeMirror div (since https://github.com/codemirror/view/commit/93f45d12293d6373c724534d64bd97cfd959cc95):
    ![Screenshot from 2023-01-02 14-21-14](https://user-images.githubusercontent.com/5832902/210244242-9804a494-28ad-4c66-bc8d-4e1d4f4aa7e9.png)
    was the accessibility test done with this CodeMirror version?
3. I see this added on code cell and markdown cell. If (1) and (2) are incorrect, then raw cell would need it too.

Reply:
Thanks @krassowski.

1. I agree that the *role* should probably be a different one for the `jp-Cell` node, especially since it contains itself a `textbox`. I'll open an issue to discuss that, the related one is on Notebook repository. 
Nevertheless, a role seems necessary on `jp-cell` node to keep the *aria-label* attribute and respect the WAI-ARIA standard.
2. Yes there is a *role* but no *aria-label*.
3. Right

Reply:
I opened https://github.com/jupyterlab/jupyterlab/issues/13704 related to discuss this point.